### PR TITLE
Ramanaj is adding a file for Cary, NC

### LINKED ISSUE #119: Add a file for ramanaj hometown
# Adding file for Cary, NC

- [ ] Visit Umstead Park
- [ ] Boating in Bond Park
- [ ] Create a branch
- [ ] Add file
- [ ] Commit my changes
- [ ] Open a pull request
- [ ] Get a code review 👍 
- [ ] Get a 👍 
- [ ] Merge the request

### PR DESCRIPTION
**Pull request by ramanaj**
This pull request resolves #119 

@thaidar Good luck with your run.

### CODE REVIEW

File: ramanaj-cary.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Cary, NC
```
Comment (byu2233) [inline ramanaj-cary.md]: Nice City

File: ramanaj-cary.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+### Great places to eat
+
+### Fun things to do
+
```
Comment (byu2233) [inline ramanaj-cary.md]: Looking forward to seeing what to do.

File: ramanaj-cary.md (hunk 1 above)
Comment (ramanaj) [inline ramanaj-cary.md]: Yes, do plan to visit sometime. Biking and Running friendly.

File: ramanaj-cary.md (hunk 2 above)
Comment (ramanaj) [inline ramanaj-cary.md]: First in Flight.

**Pull request merged**